docs/appendices/migration-guides/index.mdx

```diff
@@ -12,6 +12,43 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.18.0` -> `2.19.0`
+
+### Adding custom icons
+
+We have improved the way you can add custom icons to your theme. Before you had
+to override the
+[`theme/components/atoms/Icon/Icon.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/atoms/Icon/Icon.js)
+file to add your custom icons. Now you can simply extend or override the core
+icons with your custom icons by adding them to the
+[`theme/components/atoms/Icon/icons/app-icons.js`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/atoms/Icon/icons/app-icons.js)
+file.
+
+```js title="theme/components/atoms/Icon/icons/app-icons.js"
+import { FcHome, FcPhone } from "react-icons/fc";
+
+/** @type {Record<string, import("react-icons").IconType>} */
+const componentIcons = {
+  home: FcHome // this will override the `IoIosHome` icon from the core implementation
+  phone: FcPhone // this will add a new icon
+};
+
+/** @type {string[]} */
+const svgIcons = [];
+
+export default {
+  componentIcons,
+  svgIcons,
+};
+```
```

Comment:
**nitpick:** this is more a documentation item than a migration step.\
Customizing icons is something that is very common in FC, and I think it'd be good to have this documented anyway.

**suggestion:** create a new page in the documentation with this content, and link it from here.\
It could be:
- in [the _Advanced > Theme_ section](https://developers.front-commerce.com/docs/category/theme)
- or on [the _Essentials > Adapt the theme to your brand_ page](https://developers.front-commerce.com/docs/essentials/adapt-theme-to-your-brand) for instance

Reply:
added it 😉
![image](https://user-images.githubusercontent.com/39598117/195613402-2282a378-3aee-4108-be7e-1cff1674fda6.png)